packages/server/src/service/mg.ts

```diff
@@ -13,6 +13,20 @@ interface Options {
   force?: boolean
 }
 
+const toAddresses = (
+  emailOfficials: boolean,
+  officialEmails: string[],
+  voterEmail: string,
+  recordsEmail: string,
```

Comment:
You probably want to pass `undefined` or `null` and handle the logic separately.  Empty string being set as a 'to' in mg could generate an error.

Also, if you have more than one optional argument, you need to make the argument a single optional `Options` object with multiple optional fields.

Reply:
The MailGun API actually gets upset if I try to pass it null / undefined.  It's okay with an empty string (I tested it).  Maybe its current state is more reasonable.

Reply:
But that's good.  Passing in `null` is a programmin error -- you should just not include the email.  That MG throws an exception is a good thing: it's reminding you of this.  The semantics of empty-string are unclear.  Is it going to error out when it sends an email?  how do email clients interpret this at the recipient end?

I would explicitly write logic to remove this value -- the fact that empty-string doesn't throw an error is just a dangerous side effect.

Reply:
Okay, that being said, I think it's in a good state now.